web/templates/web/index.html

```diff
@@ -33,36 +33,50 @@
                 <div class="item">
-                    <div class="ui small clickable image" href="{% url 'event' event.event.pk %}">
+                    <div class="ui small clickable image" href="{% url 'event' event_dict.event.pk %}">
                         <p class="ui yellow make-bg-yellow ribbon label">
-                            {{ event.start_time|date:"d" }}<br>
-                            {{ event.start_time|date:"M" }}
+                            {{ event_dict.first_occurrence.start_time|date:"d M" }}
+                            {% if event_dict.number_of_occurrences > 1 %}
+                                <br/>
+                                <span>
+                                    +{{ event_dict.number_of_occurrences|add:-1 }}
+                                    {% if event_dict.number_of_occurrences > 2 %}{% trans "dates" %}{% else %}{% trans "date" %}{% endif %}
+                                </span>
+                            {% endif %}
                         </p>
-                        <img {% thumbnail event.event.image "300" crop="center" as image_thumb %}
+                        <img {% thumbnail event_dict.event.image "300" crop="center" as image_thumb %}
                             src="{{ image_thumb.url }}"
                         {% empty %}
-                            src="{{ event.event.image.url }}"
+                            src="{{ event_dict.event.image.url }}"
                         {% endthumbnail %}
-                            {% if event.event.contain %}style="object-fit: contain !important;"{% endif %}
+                            {% if event_dict.event.contain %}style="object-fit: contain !important;"{% endif %}
                         />
                     </div>
 
                     <div class="content">
                         <div class="header">
-                            <a href="{% url 'event' event.event.pk %}">{{ event.event.title }}</a>
+                            <a href="{% url 'event' event_dict.event.pk %}">{{ event_dict.event.title }}</a>
                         </div>
                         <div class="meta">
-                            <span>{{ event.place }}</span>
+                            <span>{{ event_dict.first_occurrence.place }}</span>
                         </div>
                         <div class="description">
-                            <p>{{ event.event.clickbait }}</p>
+                            <p>{{ event_dict.event.clickbait }}</p>
                         </div>
                     </div>
                 </div>
             {% endfor %}
         </div>
+        {% if more_events_exist %}
+            <h2 class="more-events-link">
+                <a href="{% url 'events' %}">
+                    {% trans "More events" %} <i class="angle double right icon"></i>
+                </a>
+            </h2>
+        {% endif %}
+        <hr class="make-bg-yellow event-article-border"/>
```

Comment:
Isn't the yellow make color already specified in `event-article-border` which makes it redundant?

Reply:
Well, the `event-article-border` class sets `border-color`, whereas `make-bg-yellow` sets `background-color`, so I wouldn't say it's redundant 😅

Reply:
Ah, then it seems that the `background-color` minimally impact the line though.